radis/io/hitran.py

```diff
@@ -407,8 +406,7 @@ def hit2df_hapi(
     """
     from hapi import LOCAL_TABLE_CACHE
 
-    data_file_name = data_file.split(".")
-    df = pd.DataFrame(LOCAL_TABLE_CACHE[data_file_name[0]]["data"])
+    df = pd.DataFrame(LOCAL_TABLE_CACHE[os.path.basename(fname).split(".")[0]]["data"])
```

Comment:
good. I was thinking of `os.path.splitext` , doesit work ?

Reply:
It actually splits the path by delimiter `.`  so we get an output = 
('C:\\Users\\Supriya\\.radisdb\\hitran\\downloads__can_be_deleted\\CO\\CO_5',
 '.data')`, we require `CO_5` so again we will need to access the 0th index and split it using `\\` and again access is -1 index, which is more work so I believe sticking to the current implementation will be better

Reply:
👍